libs/nuxt/src/migrations/update-11.0.0/update-11.0.0.ts

```diff
@@ -0,0 +1,49 @@
+import { chain, Rule } from '@angular-devkit/schematics';
+import {
+  addDepsToPackageJson,
+  readWorkspace,
+  updateJsonInTree,
+  updatePackagesInPackageJson,
+} from '@nrwl/workspace';
+import * as path from 'path';
+
+function deleteExtendInEslintConfig(): Rule {
+  return (tree) => {
+    const { projects } = readWorkspace(tree);
+    for (const key in projects) {
+      const project = projects[key];
+      if (
+        (project?.architect?.build?.builder === '@nx-plus/nuxt:browser' ||
+          project?.targets?.build?.executor === '@nx-plus/nuxt:browser') &&
+        (project?.architect?.lint || project?.targets?.lint)
+      ) {
+        const eslintConfig = '.eslintrc.js';
+        const content = tree
+          .read(path.join(project.root, eslintConfig))
+          .toString('utf-8')
+          .replace(/'prettier\/@typescript-eslint',?/g, '');
+
+        tree.overwrite(path.join(project.root, eslintConfig), content);
+      }
+    }
+  };
+}
+
+export default function update(): Rule {
+  return chain([
+    updatePackagesInPackageJson(
+      path.join(__dirname, '../../../', 'migrations.json'),
+      '11.0.0'
+    ),
+    addDepsToPackageJson(
+      { 'core-js': '^3.8.3' },
+      { 'eslint-config-prettier': '8.1.0' }
+    ),
+    updateJsonInTree('package.json', (json) => {
+      delete json.devDependencies['@nuxtjs/eslint-config'];
+      delete json.devDependencies['fork-ts-checker-webpack-plugin'];
```

Comment:
Like we discussed offline, it may be best to leave these dependencies. I think it is. It's higher priority to keep a users workspace functional than "clean".